crates/aptos-telemetry/src/sender.rs

```diff
@@ -71,7 +71,7 @@ impl TelemetrySender {
         let mut response = request_builder
             .try_clone()
             .expect("Could not clone request_builder")
-            .bearer_auth(token)
+            .bearer_auth(token.clone())
```

Comment:
why the `.clone()` ? it this necessary?

Reply:
Yup, it isn't necessary. I was using it elsewhere before for debugging so I had used clone. linter dint catch apparently.